export_presets.cfg

```diff
@@ -24,7 +24,6 @@ binary_format/embed_pck=true
 custom_template/release=""
```

Comment:
Can you revert the changes to this file? It's been set up for the auto builds, so changes shouldn't be committed to it

Reply:
Yep, sure.